templates/rhpam70-authoring-ha.yaml

```diff
@@ -1108,6 +1108,21 @@ objects:
         - name: "${APPLICATION_NAME}-mysql"
           image: mysql
           imagePullPolicy: Always
+          livenessProbe:
+            initialDelaySeconds: 30
+            tcpSocket:
+              port: 3306
+            timeoutSeconds: 1
+          readinessProbe:
+            initialDelaySeconds: 5
+            exec:
+              command:
+                - "/bin/sh"
+                - "-i"
+                - "-c"
+                - "MYSQL_PWD=\"$MYSQL_PASSWORD\" mysql -h 127.0.0.1 -u $MYSQL_USER -D $MYSQL_DATABASE -e 'SELECT 1'"
+              port: 3306
+            timeoutSeconds: 1
```

Comment:
just a little observation, to standardize, move initialDelaySeconds down the timeoutSeconds , or vice-versa, but I think the first option better. this same comment is valid for the other mysql readinessProbes.